app/concepts/cinema_halls/representers/single.rb

```diff
@@ -5,15 +5,49 @@ class Single
 
       def initialize(cinema_hall)
         @cinema_hall = cinema_hall
+        @not_available = cinema_hall[:not_available]
+        @seats = cinema_hall[:seats]
       end
 
       def basic
         {
           id: cinema_hall.id,
           name: cinema_hall.name,
-          capacity: cinema_hall.capacity
+          capacity: cinema_hall.capacity,
+          seats: seats
         }
       end
+
+      def extended
+        {
+          id: cinema_hall.id,
+          name: cinema_hall.name,
+          capacity: cinema_hall.capacity,
+          seats: seats,
+          screenings: screenings
+        }
```

Comment:
Because you used it here:
https://github.com/kodowicz/monte-cinema/blob/a88b782101936abd9718326579abea97245247cd/app/concepts/reservations/representers/single.rb#L18-L22
I will just suggest doing it here in the same way :)